### ISSUE TITLE
Cost summary auto or manual refresh using Power Bi service returning an error as unsupported

### ISSUE THREAD

**Issue by NavidKhan8** [microsoft/finops-toolkit#1102]
We are trying to setup Cost Summary PB report using the link https://microsoft.github.io/finops-toolkit/power-bi/cost-summary
FinOps Hub setup completed and .parquet data export conversion completed. From Ingestion setting.json hub version is 0.6

We are able to setup cost summary report using Power Bi desktop locally and published to Power Bi workspace but in Power Bi service of workspace unable to perform refresh and getting below error:

Error:
powerbi error this dataset includes a dynamic data source. Since dynamic data source arent' refreshed in the power bi service , this dataset wont be refreshed

Screenshot:
![Image](https://github.com/user-attachments/assets/70f9ad1f-4e11-4f26-ab83-c9bf5fc53e16)

Kindly check and confirm.

**Comment by flanakin** [comment]
Hey, Navid. I know we had this issue with some of the other reports. I didn't realize we had it with this one as well. We're about to ship 0.7. If it's okay, I would suggest using that version of the Power BI report, which is backwards compatible with FinOps hubs 0.6. Otherwise, I can look into this after FinOps X Europe.

**Comment by NavidKhan8** [comment]
[like]  Navid Ulla (External) reacted to your message:
________________________________

**Comment by flanakin** [comment]
@NavidKhan8 can you try the latest reports and let us know if you're still seeing issues? Note the file renamed to PowerBI-storage.zip. 

https://aka.ms/ftk/latest

**Comment by NavidKhan8** [comment]
@flanakin  Sure, I will try the newer version 0.7 and update you how it goes.

**Comment by arthurclares** [comment]
@NavidKhan8 Have you been able to test the latest version? Thank you!

**Comment by NavidKhan8** [comment]
> [@NavidKhan8](https://github.com/NavidKhan8) Have you been able to test the latest version? Thank you!

Yes, I have tried 0.8 and it worked fine to load 3 months data but when tried for 6 months data getting below screenshot error

![Image](https://github.com/user-attachments/assets/d65c3028-4ebd-4131-bbfd-27276cc570ba)

**Error message:**
Data source error: The XML for Analysis request timed out before it was completed. Timeout value: 18000 sec. Table: Costs.

It timed out after running for 5 hours of refresh.

**Comment by flanakin** [comment]
@NavidKhan8 This is likely due to having too much data. I would highly recommend deploying FinOps hubs with Data Explorer. When you do this, you won't need to schedule an import and the report will be updated when users open each page. Performance is in seconds instead of hours.

**Comment by NavidKhan8** [comment]
> [@NavidKhan8](https://github.com/NavidKhan8) This is likely due to having too much data. I would highly recommend deploying FinOps hubs with Data Explorer. When you do this, you won't need to schedule an import and the report will be updated when users open each page. Performance is in seconds instead of hours.

I've tried earlier and faced issues but that was 2 months ago so I will try again and get back to you with findings. Fingers crossed!

**Comment by NavidKhan8** [comment]
@flanakin - Thank you for the extended help in setting up FinOps hub using data explorer. Much appreciated!

Now I'm able to fetch the 3months data to PowerBi report in no time after exporting it to data explorer.

I will continue to export 6, 9 and 12 months data and keep you posted.
